packages/headless/src/integration-tests/commerce.test.ts

```diff
@@ -69,4 +83,16 @@ describe.skip('commerce', () => {
       ])
     );
   });
+
+  it('applies sort to product listing', async () => {
```

Comment:
Oh good call, I'll add integration tests for pagination too

Reply:
They're skipped for now, since we don't have an org with stable-enough fixtures. But they're definitely helpful as a sanity check before doing an actual implementation 😅